src/components/Capture/Face.js

```diff
@@ -24,7 +24,7 @@ const defaultPayload = {
 
 class Face extends Component {
   static defaultProps = {
-    useWebcam: true,
+    useWebcam: true,  // FIXME: remove code dependency on useWebcam once PR #762 for UI tests refactor is merged into 'development' branch
```

Comment:
The tests refactor work has been merged

Reply:
I've marked it as out of scope though as I feel the clean up for it should be a ticket by itself and should include a check that other areas of the SDK are not using it incorrectly.

Reply:
Think I need to check how the camera tests work as my attempt to remove `useWebcam` in the Face Capture step resulted in quite a few test failures. 😞

Reply:
Sure, it can be done as part of another story